source/workflowapi/app.py

```diff
@@ -501,7 +501,7 @@ def create_operation(operation):
     "StageName.$": "$.Name",
     "Name":"%%OPERATION_NAME%%",
     "Input.$":"$.Input",
-    "Configuration.$":"$.Configuration.%%OPERATION_NAME%%",
+    "g.$":"$.Configuration.%%OPERATION_NAME%%",
```

Comment:
As discussed, this change looks inadvertent.  Since this really does look like dead code, we should just remove the whole block to avoid any future confusion.